presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -182,7 +201,7 @@ public boolean isLimitGuaranteed(ConnectorSession session)
     @Override
     public Optional<JdbcTableHandle> getTableHandle(ConnectorSession session, SchemaTableName schemaTableName)
     {
-        TableHandleCacheKey key = new TableHandleCacheKey(JdbcIdentity.from(session), schemaTableName);
```

Comment:
`new TableHandleCacheKey(session, schemaTableName)`? Same for others. Or `TableHandleCacheKey.create(session, schemaTableName)`

Reply:
Latter is not possible (I'd need to pass Set<SessionPropertyProvider>)